packages/components/src/internal/components/buttons/ActionButton.spec.tsx

```diff
@@ -24,20 +24,24 @@ describe('<ActionButton />', () => {
     test('Default properties', () => {
         const onClick = jest.fn();
         const wrapper = shallow(<ActionButton onClick={onClick} />);
-        console.log(wrapper.debug());
         wrapper.find('span').simulate('click');
         expect(onClick).toHaveBeenCalledTimes(1);
         expect(wrapper).toMatchSnapshot();
     });
 
-    test('Specify entity and classes', () => {
+    test('With custom props', () => {
```

Comment:
I assume this test name change means that a new snapshot was created in `ActionButton.spec.tsx.snap` and the original one for the test name "Specify entity and classes" is now orphaned. Can you check and clean up that `ActionButton.spec.tsx.snap`?

Reply:
Oop yup, done!